SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -31,16 +33,23 @@ _log "Setting up the environment..."
 ACTIVATE_ENV_CMD="source ${ENV_PATH}/bin/activate"
 eval "${ACTIVATE_ENV_CMD}"
 
-# Install packages in the environment and the same ipykernel that the Jupyter server uses
+# If requirements have its dependencies and versions resolved,
+# install them directly with uv for better performance
+# Otherwise, use the default pip, which is slower but more reliable
```

Comment:
I think it's enough to say:
```
# Install user-requested packages in the environment
```
You already include some docs about this topic when you define `RESOLVED_REQ`, right?